ts/store/actions/legacyWallet.ts

```diff
@@ -0,0 +1,28 @@
+import { ActionType, createAsyncAction } from "typesafe-actions";
+import { RptId } from "@pagopa/io-pagopa-commons/lib/pagopa";
+import { PaymentRequestsGetResponse } from "../../../definitions/backend/PaymentRequestsGetResponse";
+import { Detail_v2Enum as PaymentProblemErrorEnum } from "../../../definitions/backend/PaymentProblemJson";
+
+export type PaymentStartOrigin =
+  | "message"
+  | "qrcode_scan"
+  | "poste_datamatrix_scan"
+  | "manual_insertion"
+  | "donation";
+//
+// verifica
+//
+
+// the error is undefined in case we weren't able to decode it, it should be
+// interpreted as a generic error
+export const paymentVerifica = createAsyncAction(
```

Comment:
Yes, this was intentional. This action is inherited from the same functionality mentioned above and is also used in other streams such as Communication & Messages. To avoid renaming it and impacting files in other domains, I'd prefer to keep it as is, especially since it’s the only legacy functionality that still needs to remain in the app until next month.

Reply:
Thanks for the clarifications!